python/tools/REx_calibrate_trajectory_limits.py

```diff
@@ -0,0 +1,1918 @@
+#!/usr/bin/python3
+from dataclasses import asdict, dataclass, field
+from enum import IntEnum
+import json
+import os
+import subprocess
+
+from stretch_body.hello_utils import *
+import argparse
+import click
+
+from stretch_body.pimu import Pimu
+from stretch_body.lift import Lift
+from stretch_body.arm import Arm
+from stretch_body.base import Base
+from stretch_body.prismatic_joint import PrismaticJoint
+
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+
```

Comment:
I found it necessary to add this here:
```
os.environ['QT_QPA_PLATFORM_PLUGIN_PATH'] = '/usr/lib/x86_64-linux-gnu/qt5/plugins/platforms/libqxcb.so'
```

Without the above fix, I'd get QT error during the first calibration step.